daemon/container_windows.go

```diff
@@ -193,6 +185,7 @@ func (container *Container) PrepareStorage() error {
 	return nil
 }
 
+// CleanupStorage unprepares the layer after shutdown? FIXME
```

Comment:
@jhowardmsft May I ask for a comment on this method, and perhaps a review of the rest of this file as well?